akka-remote/src/main/scala/akka/remote/transport/netty/NettyTransport.scala

```diff
@@ -186,6 +186,25 @@ class NettyTransportSettings(config: Config) {
       config.getDouble("pool-size-factor"),
       config.getInt("pool-size-max"))
 
+  // Check Netty version >= 3.10.6
+  {
+    val nettyVersion = org.jboss.netty.util.Version.ID
+    def throwInvalidNettyVersion(): Nothing = {
```

Comment:
throwIfNettyVersionInvalid

Reply:
well, the `if` is not inside this `def`
I extracted it because used from two places

Reply:
Oh,yes!